src/window/index.ts

```diff
@@ -58,27 +63,58 @@ function createWindow(name: string, url: string, prefs: Electron.WebPreferences)
 
     return { action: 'deny' };
   });
+  window.webContents.on('did-fail-load', (event, errorCode, errorDescription, url) => {
+    console.log(`Failed to load ${ url }: ${ errorCode } (${ errorDescription })`);
+  });
   window.loadURL(url);
   windowMapping[name] = window.id;
+
+  return window;
 }
 
 /**
  * Open the preferences window; if it is already open, focus it.
  */
 export function openPreferences() {
-  let url = 'app://./index.html';
+  const webRoot = getWebRoot();
```

Comment:
Could we have `getWebRoot` not return a trailing slash, so consumers have to add one? I find a line like `${ webRoot }/index.html` to be clearer to read.

Reply:
Fixed in f717e7b870c177608a4fcb5a4daf3bc6f7ac8aed